### PR TITLE
feat: Live update transaction details

### LINKED ISSUE #543: Live update transaction details
Transaction details should be updated live (Keyblock confirmations).

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #543

## Demo

https://github.com/aeternity/aescan/assets/15363559/e7d7268c-9942-43cd-8e26-ba99c9869a9b

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] make confirmations count live

### CODE REVIEW

File: src/stores/webSocket.js [hunk 1]
```diff
@@ -6,13 +6,25 @@ import { useTransactionDetailsStore } from '@/stores/transactionDetails'
...
-  const { processSocketMessage } = useRecentBlocksStore()
+  const { processSocketMessage, updateBlockHeight } = useRecentBlocksStore()
   const { updateTransactionTypeData } = useTransactionDetailsStore()
 
   const webSocket = shallowRef()
+  const isSubscribedToTransactionDetails = ref(false)
   const isSubscribedToKeyblockDetails = ref(false)
   const subscribedTransactionId = ref(null)
 
+  watch(isSubscribedToTransactionDetails, newValue => {
```
Comment (michele-franchi) [inline src/stores/webSocket.js:17]: Is this logic needed? Can't we just leave the subscription open?
When the webSocket is opened the subscription is already triggered:
https://github.com/aeternity/aescan/pull/569/files#diff-6eaf0a1b5ea40a2f808855772c29c950791df43f5d4e386e7e89d8cb3542589bR66
Could this cause some issues?

**Review by michele-franchi — APPROVED** [review]
It works good but I think some logic is over complicated. Is it really needed to close Keyblock and Microblocks subscriptions in different cases since for example it's not specific to a particular transaction?

File: src/stores/webSocket.js (hunk 1 above)
Comment (janmichek) [reply src/stores/webSocket.js]: Yep, you are right. Can be simplified .) Thanks for the suggestion. Fixed

Comment (michele-franchi) [reply src/stores/webSocket.js]: Looks good 👏

**Pull request merged**